biosys/apps/main/models.py

```diff
@@ -305,6 +305,14 @@ def __str__(self):
         return self.code
 
 
+@python_2_unicode_compatible
+class Form(models.Model):
+    name = models.CharField(max_length=200, null=False, blank=False)
+    layout = JSONField()
+
+    def __str__(self):
+        return '{}'.format(self.name)
+
```

Comment:
where is the relationship with the Dataset?

Reply:
i was in so much of a rush I forgot, I will but put it in now